sql_generators/mobile_kpi_support_metrics/templates/attribution_clients.query.sql

```diff
@@ -0,0 +1,240 @@
+{{ header }}
+WITH new_profiles AS (
+  SELECT
+    submission_date,
+    client_id,
+    sample_id,
+    {% if 'is_suspicious_device_client' in product_attribution_group_names %}
```

Comment:
I see it on the table but not on the view

Reply:
You are correct. I added it now.